### PR TITLE
Fix parity between QuestsPage and DiscoveriesPage

### PR DESCRIPTION
**Pull request by Incompleteusern**
See https://discord.com/channels/394189072635133952/424990854290210837/874717069717954561

### CODE REVIEW

File: src/main/java/com/wynntils/modules/questbook/overlays/ui/QuestsPage.java [hunk 1]
```diff
@@ -241,9 +241,10 @@ protected boolean isHovered(int index, int posX, int posY) {
         }
 
         if (entryInfo.hasTargetLocation()) {
-            lore.add(TextFormatting.YELLOW + (TextFormatting.BOLD + "Middle click to view on map!"));
+            lore.add(TextFormatting.YELLOW + (TextFormatting.BOLD + "Right click to view on map!"));
         }
-        lore.add(TextFormatting.GOLD + (TextFormatting.BOLD + "Right click to open on the wiki!"));
+
+        lore.add(TextFormatting.GOLD + (TextFormatting.BOLD + "Middle click to open on the wiki!"));
```
Comment (HeyZeer0) [inline src/main/java/com/wynntils/modules/questbook/overlays/ui/QuestsPage.java:247]: Useless space

File: src/main/java/com/wynntils/modules/questbook/overlays/ui/QuestsPage.java [hunk 2]
```diff
@@ -241,9 +241,10 @@ protected boolean isHovered(int index, int posX, int posY) {
         }
 
         if (entryInfo.hasTargetLocation()) {
-            lore.add(TextFormatting.YELLOW + (TextFormatting.BOLD + "Middle click to view on map!"));
+            lore.add(TextFormatting.YELLOW + (TextFormatting.BOLD + "Right click to view on map!"));
```
Comment (HeyZeer0) [inline src/main/java/com/wynntils/modules/questbook/overlays/ui/QuestsPage.java:244]: I feel like right-click to go to the wiki seems more useful? Have you done a pool to check which way is preferred? 🤔

Comment (Incompleteusern) [reply src/main/java/com/wynntils/modules/questbook/overlays/ui/QuestsPage.java:244]: How would I make such a pool? Also true, my original reason is wrong

Comment (magicus) [reply src/main/java/com/wynntils/modules/questbook/overlays/ui/QuestsPage.java:244]: @Incompleteusern You can make a simple poll in Discord. Ask the question and give yourself an arrow-up and arrow-down emoji, and ask people to click on the one they agree with.

**Pull request closed without merging**